config/locales/en/common.yml

```diff
@@ -4,6 +4,7 @@ en:
     'no': 'No'
     edit: 'Edit'
     delete: 'Delete'
+    delete_confirm: 'Are you sure to delete?'
```

Comment:
I believe there is a way to use the default ActiveRecord translations for operations like this, like how SimpleForm can generate the Update/Create messages without us specifying a default value.